flowkit_jwt_generator/flowkit_jwt_generator/fixtures.py

```diff
@@ -0,0 +1,120 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Provides several convenient fixtures for working with JWTs in pytest
+#
+
+import os
+from datetime import timedelta
+from typing import Dict, Union, List, Callable, Optional
+
+import pytest
+from cryptography.hazmat.backends.openssl.rsa import _RSAPrivateKey, _RSAPublicKey
+from cryptography.hazmat.primitives import serialization
+
+from flowkit_jwt_generator import (
+    load_private_key,
+    load_public_key,
+    get_all_claims_from_flowapi,
+    generate_token,
+)
+
+
+@pytest.fixture
+def private_key() -> _RSAPrivateKey:
+    """
+    Fixture which returns a private key loaded from a PRIVATE_JWT_SIGNING_KEY env var.
+    """
+    return load_private_key(os.environ["PRIVATE_JWT_SIGNING_KEY"])
+
+
+@pytest.fixture
+def public_key() -> _RSAPublicKey:
+    """
+    Fixture which returns a public key loaded from a PUBLIC_JWT_SIGNING_KEY env var.
+    """
+    return load_public_key(os.environ["PUBLIC_JWT_SIGNING_KEY"])
+
+
+@pytest.fixture
+def public_key_bytes(public_key) -> bytes:
+    """
+    Fixture which returns the PEM encoded bytestring representation of a  public key loaded
+    from a PUBLIC_JWT_SIGNING_KEY env var.
+    """
+    return public_key.public_bytes(
+        encoding=serialization.Encoding.PEM,
+        format=serialization.PublicFormat.SubjectPublicKeyInfo,
+    )
+
+
+@pytest.fixture
+def private_key_bytes(private_key) -> bytes:
+    """
+    Fixture which returns the PEM encoded bytestring representation of a private key loaded
+    from a PRIVATE_JWT_SIGNING_KEY env var.
+    """
+    return private_key.private_bytes(
+        encoding=serialization.Encoding.PEM,
+        format=serialization.PrivateFormat.TraditionalOpenSSL,
+        encryption_algorithm=serialization.NoEncryption(),
+    )
+
+
+@pytest.fixture
+def access_token_builder(audience: Optional[str] = None) -> Callable:
+    """
+    Fixture which builds short-life access tokens.
+
+    Parameters
+    ----------
+    audience : str, default None
```

Comment:
What's an example for the `audience` string? Also, all the usages I can find in the code seem to pass a dict instead of a string?

Reply:
Never mind, the dict of course contains the claims passed to the `token_maker` closure below. Would still be useful to have an example of the `audience` string here.